scripts/firefox/jumpcloud/windows/v1/zluri-windows-firefox-removal-jumpcloud.ps1

```diff
@@ -38,6 +38,7 @@ if (Test-Path $profilesPath) {
                                 # Stop all Firefox processes
                                 Stop-Process -Name firefox -Force -ErrorAction Stop
                                 Write-Output "Successfully terminated all Firefox processes."
+                                Start-Sleep -Seconds 1
```

Comment:
@sashank-zluri  I was testing one case where there are more than 10 tabs open on firefox
For that process to kill, we need little time to added this delay.